releasenotes/notes/add-grover-primitives-10f81efdba93703d.yaml

```diff
@@ -0,0 +1,21 @@
+---
+features:
+  - |
+
+    :class:`~.Grover` supports the primitives and can use :class:`~.BaseSampler` to
+    calculate the results. Accordingly, ``quantum instance`` in :class:`~.Grover`
+    is pending deprecation and will be deprecated in a future release.
+
+    Example::
+    .. code-block:: python
```

Comment:
This will need correcting as was done for some existing already merged release notes in #8777

I get that its a skeleton here but would it make sense to have it be a small complete working example that you could cut/paste and run it. We have this in other places where it has the imports and everything

Reply:
I thought it's merged 🤣
Alright, I changed the code-block and now it's an actual code that can work.